libs/shell/controls/src/lib/controls-shell.module.ts

```diff
@@ -28,17 +30,28 @@ import { SlideToggleComponent } from "./ui/slide-toggle/slide-toggle.component";
     MatInputModule,
     MatListModule,
     MatSelectModule,
+
```

Comment:
I think that linter is adding these lines..

Reply:
Isn't the linter, is prettier. I'll work on this soon